api/src/test/java/org/openmrs/api/ProgramWorkflowServiceUnitTest.java

```diff
@@ -76,40 +75,34 @@ public void getProgramByName_shouldFailWhenTwoProgramsFoundWithSameName() {
 		Mockito.when(mockDao.getProgramsByName("A name", false)).thenReturn(programsWithGivenName);
 		Mockito.when(mockDao.getProgramsByName("A name", true)).thenReturn(programsWithGivenName);
 		pws.setProgramWorkflowDAO(mockDao);
-		pws.getProgramByName("A name");
+		
+		Assertions.assertThrows(ProgramNameDuplicatedException.class,() ->pws.getProgramByName("A name"));
```

Comment:
I know in the tests you are migrating we are not always using static imports for all assertions. You can statically import them tough as it improves readability. Not sure which IDE you are using but with IntelliJ for example it almost does it for you. But since this is a huge task I also understand its not the most important thing. So feel free to ignore this.